geopandas/io/arrow.py

```diff
@@ -48,6 +48,24 @@ def _is_fsspec_url(url):
     )
 
 
+def _remove_id_from_member_of_ensembles(json_dict):
+    """
+    Older PROJ versions will not recognize IDs of datum ensemble members that
+    were added in more recent PROJ database versions.
+
+    Cf https://github.com/opengeospatial/geoparquet/discussions/110
+    and https://github.com/OSGeo/PROJ/pull/3221
+
+    Mimicking the patch to GDAL from https://github.com/OSGeo/gdal/pull/5872
+    """
+    for key, value in json_dict.items():
+        if isinstance(value, dict):
+            _remove_id_from_member_of_ensembles(value)
+        elif key == "members" and isinstance(value, list):
+            for member in value:
+                member.pop("id")
```

Comment:
```suggestion
                member.pop("id", None)
```

or 

```Python
if "id" in member:
    member.pop("id")
```


To avoid the `KeyError` we're seeing on some of the CI runs.

Reply:
That's what you get if you don't run the tests locally ... Thanks!